Docker/itksoftwareguide-bin/README.md

```diff
@@ -1 +1,21 @@
+Description
+-----------
+
 Holds the volume with the build for the Software Guide.
+
+Volumes
+-------
+
+Source folder: /home/itk/src/ITKSoftwareGuide
+Binary folder: /home/itk/bin/ITKSoftwareGuide
+
+Usage
+-----
+
+Build in the docker container:
+  ./run.sh itksoftwareguide-bin
+
+Use ITKSoftwareGuide outside of the docker container:
+  ./run.sh itksoftwareguide-bin \
+    -v ${HostITKSoftwareGuidSourceDirectory}:/home/itk/src/ITKSoftwareGuide \
+    -v ${HostITKSoftwareGuidBinaryDirectory}:/home/itk/bin/ITKSoftwareGuide
```

Comment:
@thewtex : I updated the documentation following the code update I did which allows to pass parameters to `docker` through the script `run.sh`. Please review that part too as it is one of the two main improvements of this patch (with building `CMake` with OpenSSL support)